anaconda_linter/lint/check_completeness.py

```diff
@@ -160,21 +160,36 @@ def check_recipe(self, recipe_name: str, arch_name: str, recipe: RecipeReaderDep
         self._validate_if_recipe_path_is_missing("/about/license_file", validate_value_str)
 
 
-class license_file_overspecified(LintCheck):
+class deprecate_license_url(LintCheck):
     """
-    Using license_file and license_url is overspecified.
-
-    Please remove license_url.
+    This field is deprecated in V1 recipes.
+    Please remove license_url
     """
 
-    def check_recipe_legacy(self, recipe: Recipe) -> None:
-        if recipe.get("about/license_file", "") and recipe.get("about/license_url", ""):
-            self.message(section="about", severity=Severity.WARNING, data=recipe)
+    def check_recipe(self, recipe_name: str, arch_name: str, recipe: RecipeReaderDeps) -> None:
+        recipe = self.recipe
+        if recipe.contains_value("/about/license_url"):
+            self.message(section="/about/license_url", severity=Severity.ERROR)
+        output_paths = recipe.get_package_paths()
+        for package_path in output_paths:
+            if package_path == "/":
+                continue
+            path: Final = recipe.append_to_path(package_path, "/about/license_url")
+            if recipe.contains_value(path):
+                self.message(section=path, severity=Severity.ERROR)
 
     def fix(self, message, data) -> bool:
-        recipe = data
-        op = [{"op": "remove", "path": "/about/license_url"}]
-        return recipe.patch(op, op_mode=OpMode.PARSE_TREE)
+        section = message.section
+        recipe = self.unrendered_recipe
+        if recipe is None:
+            return False
+        path: Final = section if section else "/about/license_url"
+        op = {
+            "op": "remove",
+            "path": path,
+            "value": "",
+        }
+        return recipe.patch(op)
```

Comment:
Personally, I would just put the patch dictionary in-place and ditch the variable:

```suggestion
        return recipe.patch({
            ...
        })
```